athf/mcp/server.py

```
@@ -51,10 +59,14 @@ def create_server(workspace_path: Optional[str] = None) -> "FastMCP":  # type: i
-    except ImportError:
-        raise ImportError(
-            "MCP dependencies not installed. Install with: pip install 'athf[mcp]'"
-        ) from None
+    except ImportError as exc:
+        raise MCPDependencyError(
+            f"Could not import mcp.server.fastmcp.FastMCP ({exc}). This usually "
+            "means either the mcp package is not installed, or an incompatible "
+            "version is installed: mcp 2.0.0 removed FastMCP. Install a supported "
+            "version with: pip install 'athf[mcp]' (which pins "
+            "mcp[cli]>=1.9.4,<2.0.0)."
+        ) from exc
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)